src/api-guides/payment-provider-integration.html

```diff
@@ -0,0 +1,109 @@
+
+### <a name="Topics-to-complete-prior-to-obtaining-invoices"></a>Topics to complete prior to obtaining invoices
+
+**These topics should be completed prior to obtaining Invoices pending payment:**
+
+#### Topic 1: Update the field called, "Payment Method Type" within the SAP-Concur Invoice Vendor record
+
+In order to pay a customer's invoices, the SAP-Concur Invoice Vendor record must have the value of **PAYPVD** populated within the "Payment Method Type" field.  This can be populated in 3 ways, one of which is via API.  The Partner and customer need to agree who will be responsible for updating the Vendors initially and on an on-going basis. Here are the methods the Vendor (aka Supplier) record can be updated:
+
+Option|Technique|Owner responsible for update
+---|---|---
+Option 1|File import-On-Demand Import via an Excel template within the SAP-Concur Invoice product|Customer completes
+Option 2|File import-Scheduled Import via a csv file|Customer completes
+Option 3|App integration|Customer and Partner need to coordinate since Vendor master data most likely resides in their ERP
+
+### <a name="Update-Vendor-Payment-Method-field"></a>Update Vendor Payment Method field
+
+[Vendor Update API v3](/api-reference/invoice/v3.vendor.html#put)
+
+#### Topic 2: Customer ensures the Vendor Contact Email addresses exist within the SAP-Concur Invoice Vendor record for the Vendors that the Partner will pay
+
+Confirm that the customer adminsitrator has created a Concur Audit Rule to require the Vendor (Supplier) email address has been populated for those Vendor invoices that will be paid via this integration. The Customer can take the following steps to create the Audit Rule:
+
+#### Audit Rule step 1:
+Navigate to the Administration menu, then Invoice, then Audit Rules.
+Click New
+
+Name: Vendor Supplier Contact Email
+Event: Payment Request Submit
+Editable By: this depends on the customer configuration
+Applies To: this depends on the customer configuration
+Active: Yes
+Click Next
+
+#### Audit Rule step 2: Create Condition
+
+Data Object: Vendor Remittance Address
+Field: Contact Email
+Operator: Is Blank
+Operator: And
+Data Object: Request
+Field: Pay Method Type
+Operator: Equal
+Value: Payment Provider
+
+Click Next
+
+#### Audit Rule step 3:
+The Exception Level can be adjusted if desired.  99 is a hard stop rule.
+
+Click New
+
+Exception Code: EMAIL
+Exception Level: 99
+Message: The Vendor Email Address is required for all invoices that are from Vendors that will be paid by the Partner.
+Editable By: this depends on the customer configuration
+
+### <a name="Overview-Scopes-and-Obtaining-Invoices-pending-payment"></a>Overview Scopes and Obtaining Invoices pending payment
+
+[Overview, Scopes, an Obtaining Invoices pending payment] (./invoice/v1.invoice-pay.html#overview)
+
+### <a name="Discounts"></a>Discounts on invoices
+
+[Payment Request ID](/api-reference/invoice/payment-request.html#get) and (/api-explorer/v3-0/PaymentRequest.html)
```

Comment:
You always have to pair []() together -- so this is broken syntax at the moment for the second link.